src/fides/api/api/v1/endpoints/user_endpoints.py

```diff
@@ -591,39 +609,64 @@ def user_login(
     )
 
 
-def perform_login(
-    db: Session,
-    client_id_byte_length: int,
-    client_secret_btye_length: int,
-    user: FidesUser,
-) -> ClientDetail:
-    """Performs a login by updating the FidesUser instance and creating and returning
-    an associated ClientDetail.
+async def verify_invite_code(
```

Comment:
nice to include this via dependency injection! is there a particular reason it's `async`?  just seems a bit odd especially since a non-async db `Session` is used and there aren't any actual async operations being `await`ed in the function.